packages/transaction-controller/src/types.ts

```diff
@@ -43,6 +44,7 @@ type TransactionMetaBase = {
   blockNumber?: string;
   deviceConfirmedOn?: WalletDevice;
   verifiedOnBlockchain?: boolean;
+  txReceipt?: TransactionReceipt;
```

Comment:
As this is the first property we're adding, should we sort the properties and the JSDoc in alphabetical order for easier maintenance going forward?

Reply:
I'm wondering if we should also move the descriptions inline also to be attached to the properties themselves for consistency?

Reply:
I was actually doing the same comment on [Vini's PR](https://github.com/MetaMask/core/pull/1582#discussion_r1296856733), but also moving them inline make sense to me. I will wait Vini to merge his PR then I will sort it out.